.github/workflows/e2e.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       SYSTEM_TEST_VERBOSE_OUTPUT: "true"
 
       # the soroban test cases will compile various contracts from the examples repo
-      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH: "782458d31395b0191ac40c9572ac53f13f573aa3"
+      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH: "main"
```

Comment:
Is this temporary? Could we move to using tagged releases?

Reply:
I think we need to use `main`, because that's the behavior of `init` command. If we pin the hash, we need to also do it in the `init` command, otherwise our tests won't catch the regression

Reply:
Oh good point, I feel like main is always dangerous even for init. But since we use it there we should test it.